ci_framework/roles/cifmw_cephadm/tasks/dashboard/configure_dashboard_backends.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Get the current mgr
+  command: |
+    {{ cifmw_cephadm_container_cli }} ps -a -f 'name=ceph-?(.*)-mgr.*' --format \{\{\.Command\}\} --no-trunc
+  register: ceph_mgr
+  become: true
+  delegate_to: "{{ dashboard_backend }}"
+
+- name: Check the resulting mgr container instance
+  set_fact:
+    current_mgr: "{{ ceph_mgr.stdout | regex_replace('^-n mgr.(.*)(?P<inst>) -f (.*)+$', '\\1') }}"
+
+- name: Check the resulting mgr container instance
```

Comment:
I see the variable is used other tasks like bootstrap , apply_spec and post.yaml, so i can't remove the variable but if this task is not needed , we can remove this in next iteration

Reply:
sure, +1 to keep the variable and the task as well, it might be useful for debugging purposes in case we hit issues resolving the right mgr process name.